packages/flame_jenny/README.md

```diff
@@ -0,0 +1,6 @@
+# flame_jenny
+
+Bridge package for `jenny`.
```

Comment:
Maybe add a sentence here explaining what yarn spinner is too, since this is what will show up on pub.dev

Reply:
I have an expanded version of README in PR #2113 (which is in the future from this one).